--- joshmarshall-jsonrpclib-283a2a9/jsonrpclib/jsonrpc.py	2010-09-10 09:14:38.000000000 +0200
+++ joshmarshall-jsonrpclib-283a2a9-ssl_patched/jsonrpclib/jsonrpc.py	2010-10-07 15:25:07.000000000 +0200
@@ -102,9 +102,8 @@ def jloads(json_string):
 class ProtocolError(Exception):
     pass
 
-class Transport(XMLTransport):
-    """ Just extends the XMLRPC transport where necessary. """
-    user_agent = config.user_agent
+class TransportMixIn:
+    """ Common methods for both Transport and SafeTransport. """
 
     def send_content(self, connection, request_body):
         connection.putheader("Content-Type", "application/json-rpc")
@@ -127,11 +126,15 @@ class Transport(XMLTransport):
                 print 'body: %s' % response
         return response_body
 
-class SafeTransport(XMLSafeTransport):
+class Transport(TransportMixIn, XMLTransport):
+    """ Just extends the XMLRPC transport where necessary. """
+    user_agent = config.user_agent
+
+class SafeTransport(TransportMixIn, XMLSafeTransport):
     """ Just extends for HTTPS calls """
     user_agent = Transport.user_agent
-    send_content = Transport.send_content
-    _parse_response = Transport._parse_response
+    #send_content = Transport.send_content
+    #_parse_response = Transport._parse_response
 
 class ServerProxy(XMLServerProxy):
     """
